frontend/packages/console-shared/src/components/terminal/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default as Terminal } from './Terminal';
```

Comment:
No need to export an index file since this is a one time use component and probably should not be in shared.